tests/smoke/main.rs

```diff
@@ -17,5 +17,7 @@ cfg_if::cfg_if! {
         mod notion_fetch;
         mod notion_install;
         mod autodownload;
+        #[macro_use] // to use the assert_that! macro
+        extern crate hamcrest2;
```

Comment:
Same comment here about `extern crate` and `#[macro_use]`

Reply:
will change this too